packages/react-strict-dom/tests/__snapshots__/css-test.native.js.snap-native

```diff
@@ -28,14 +28,6 @@ exports[`properties: custom property css.defineVars: tokens 1`] = `
 
 exports[`properties: custom property handles undefined variables 1`] = `{}`;
 
-exports[`properties: custom property var with hover styles 1`] = `
```

Comment:
Looks like this test was accidentally deleted?

Reply:
The test wasn't deleted, it was just changed to check the value directly instead of doing it via a snapshot

As an aside: I REALLY don't like snapshot testing when it isn't absolutely necessary — those tests (at least semantically) only verify that the testcase's output hasn't changed instead of validating that the result is actually correct.

Reply:
The value of the snapshots is that they capture the entire style object (or props), which has helped catch regressions that direct assertion of values missed.

Reply:
Okay I agree with the intent of using them but am still not convinced with how it actually works — particularly the fact that the expected results are in a different file with no clear links to the test which generated it (you can figure it out but it's not glanceable). Would there be any concerns going with [inline snapshots](https://jestjs.io/docs/snapshot-testing#inline-snapshots) instead?

Reply:
IIRC inline snapshots don't work with prettier 3. That's the error that gets printed.

In practice the tests are always run from the CLI, and can have snapshot-specific labels. Hasn't been a problem figuring out which tests are failing for StyleX or RSD so far. Furthermore, most of the tests run across both platforms so need to be external snapshots, and that makes all test snapshots consistent within the repo.